packages/components/src/utils/log/warnThatDeprecatedPropIsUsed.ts

```diff
@@ -0,0 +1,10 @@
+import { getTagNameWithoutPrefix } from '../tag-name';
+
+// TODO: add missing unit test
+export const warnThatDeprecatedPropIsUsed = (host: HTMLElement, prop: string): void => {
```

Comment:
Please rename to `warnIfDeprecatedPropIsUsed` to match other utils.

Reply:
fixed it